recipes/boost/all/conandata.yml

```
@@ -161,6 +161,10 @@ patches:
     - patch_file: "patches/1.69.0-locale-no-system.patch"
       patch_description: "This library links to boost_system, even though that library is header-only"
       patch_type: "conan"
+    - patch_file: "patches/1.76.0-0001-fix-include-inside-boost-namespace.patch"
+      patch_description: "Fix #include inside boost namespace"
+      patch_type: "official"
```

This looks like it qualifies as a `backport` patch instead of an `official` one. The `official` are mentioned in the release notes like [here](https://www.boost.org/users/history/version_1_80_0.html) for example.

```suggestion
      patch_type: "backport"
```

---

Sorry, it should be `bugfix` then right? According to the descriptions this seems to make the most sense.

---

maybe yes

---

@prince-chrismc PTAL

---

💯 a bugfix since it's exactly https://github.com/boostorg/math/commit/1ec5c98d80de97f9e962c5627e1a0e6096099894